flytekit/sdk/tasks.py

```diff
@@ -87,7 +87,7 @@ def my_task(wf_params, out1, out2):
     :rtype: flytekit.common.tasks.sdk_runnable.SdkRunnableTask
     """
     def apply_outputs_wrapper(task):
-        if not isinstance(task, _sdk_runnable_tasks.SdkRunnableTask):
+        if not isinstance(task, _task.SdkTask):
```

Comment:
This will be cleaned up eventually anyways - I'd rather not use the base class here, for now.  Can you just add the notebook tasks to this condition instead?

Reply:
done, also some other minor changes .. PTAL